jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
```

Comment:
I would have separate this commit to a different PR to ease the merging of this.

Reply:
It is in separate commit